android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDataRepository.java

```diff
@@ -472,6 +472,7 @@ public Completable saveLocations(final List<Microlocation> locations) {
     //Session Types Section
 
     private void saveSessionTypesInRealm(List<SessionType> sessionTypes) {
+        Realm realm = Realm.getDefaultInstance();
         realm.executeTransaction(realm1 -> realm1.insertOrUpdate(sessionTypes));
```

Comment:
The solution is to not create a new instance. See other method bodies on how it is done